src/pages/Dashboard/TeamOverview.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { Box, Button, Center, Spacer, Heading } from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: Int!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const StorefrontButton = () => {
+  return <Button w="199px">Share Storefront</Button>;
+};
+
+const SalesInfo = props => {
+  return (
+    <Box mr="100px">
+      <Heading fontSize="h4" color="gray.400" mb="8px">
```

Comment:
Nit: You could do `<Heading as="subtitle" size="subtitle">` to use the default heading styles (which have a font size already set)